test/tap/shrinkwrap-no-resolved.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+var test = require('tap').test
+var Tacks = require('tacks')
+var File = Tacks.File
+var Dir = Tacks.Dir
+var path = require('path')
+var common = require('../common-tap.js')
+
+var testdir = path.resolve(__dirname, path.basename(__filename, '.js'))
+
+var fixture = new Tacks(Dir({
+  'package.json': File({
+    name: 'x',
+    version: '1.0.0',
+    dependencies: {
+      '@blarg/foo': 'file://./bfoo',
+      'foo': 'file://./foo'
+    }
+  }),
+  'npm-shrinkwrap.json': File({
+    dependencies: {
+      '@blarg/foo': {
+        version: '1.0.0',
+        from: '@blarg/foo@1.0.0'
+      },
+      'foo': {
+        version: '1.0.0',
+        from: 'foo'
```

Comment:
So if you take out the `@blarg/foo` you'll see that `npm` tries to install this from the network even though it's a local file. Similarly if you have a `github` shortcut, eg `foo/bar`, here, it'll also use the network inappropriately.